app/apollo/resolvers/resource.js

```diff
@@ -35,8 +35,9 @@ const commonResourcesSearch = async ({ models, org_id, searchFilter, limit, quer
       .limit(limit)
       .lean()
     ;
+    var count = await models.Resource.find(searchFilter).count();
```

Comment:
would we be able to just do `count = resources.length` rather than making a second call into the Resources collection?

Reply:
The returned results are what we have after the limit is applied, so theyre capped at a certain number. For `count` we want to see the total number of matches in the db.